lib/controller.js

```diff
@@ -497,6 +510,7 @@ controller.provisionTessel = function(opts) {
 };
 
 controller.deployScript = function(opts) {
+  opts.authorized = true;
```

Comment:
It's not great that we have this `opts.authorized = true` all over the place. I understand that you didn't want to make it an option for each command b/c the user doesn't need access to it. But perhaps we can place this line in [`callControllWith`](https://github.com/tessel/t2-cli/blob/8848a73c1c0b0d27c2aa291e8390c2e0e4200b36/bin/tessel-2.js#L34)?